src/graphql/resolvers/global-descriptor-resolvers.ts

```diff
@@ -0,0 +1,160 @@
+import { IResolvers } from 'graphql-tools';
+import services from '../../services';
+import { IContext } from '../context';
+import { PubSub } from 'graphql-subscriptions';
+import { GlobalDescriptorService } from '../../services/global-descriptor-service';
+
+export interface IDescriptor {
+	unit: String;
+	value: number;
+	longitude: number;
+	latitude: number;
+	unixTimestamp: number;
+}
+
+export enum TRANSACTION_TYPE {

```

Comment:
Reuse

Reply:
? TRANSACTION_TYPE

Reply:
Yes, I define TRANSACTION_TYPE in my file, so why not reuse that by importing it.